cmd/mps-control-daemon/mps/daemon.go

```diff
@@ -143,6 +141,26 @@ func (d *Daemon) Start() error {
 	return nil
 }
 
+func setSELinuxContext(path string, context string) error {
+	_, err := os.Stat("/sys/fs/selinux")
```

Comment:
When comparing this to the code that is included in the `go-selinux` pacakge, it is much simpler. I assume this is sufficient for MOST cases for the time being and we can revisit if we find this check too restrictive.

Reply:
It would also be good to undestand why we would need `RW` access to this path (as required by `go-selinux`) instead of assuming that RO access is sufficient as we do here.

Reply:
I guess https://danwalsh.livejournal.com/73099.html explains it.

Reply:
I believe checking that the path exists actually means checking that SELinux is _present_ in the system.

Reply:
The `go-selinux` code aligns with the explanation from https://danwalsh.livejournal.com/73099.html. `go-selinux` always returns disabled unless a `selinuxfs` is mounted RW at `/sys/fs/selinux`.

Simply checking that the file exists aligns with what we do in the driver container and does not require any additional mounts, so I am fine with this approach. https://github.com/NVIDIA/gpu-driver-container/blob/24.07.15/rhel8/nvidia-driver#L491-L497